config/application.properties

```diff
@@ -0,0 +1,12 @@
+#Bridge related settings
+bridge.id=my-bridge
+
+#MQTT server common
+mqtt.server.host=localhost
+mqtt.server.port=1883
+
+#Apache Kafka common
+kafka.bootstrap.servers=localhost:9092
+
+#Apache Kafka producer
+kafka.producer.acks=1
```

Comment:
Other than the comment I made around the acks constant you declared in the Kafka configuration, I think that we should remove this for now from everywhere.
We have to think how to deal with making a relationship between the MQTT QoS and the Kafka acks.
It could even be something more dynamic and not static configured through the file.
But we should think more about it, not now.

Reply:
should I remove KafkaProducerConfig from this PR?
I think it should be good to keep it.

Reply:
Keep the `KafkaProducerConfig` class but remove this configuration line here for now.

Reply:
Done